lxd/network/acl/acl_ovn.go

```diff
@@ -1042,7 +1043,23 @@ type ovnLogEntry struct {
 }
 
 // ovnParseLogEntry takes a log line and expected ACL prefix and returns a re-formated log entry if matching.
-func ovnParseLogEntry(input string, prefix string) string {
+// The 'timestamp' string is in microseconds format. If empty, the timestamp is extracted from the log entry.
+func ovnParseLogEntry(input string, timestamp string, prefix string) string {
```

Comment:
please rename `timestamp` argument to `syslogTimestamp` so its clear that this should only be populated when the logLine is from syslog.